modules/telemetry/telemetrymodule.h

```diff
@@ -0,0 +1,182 @@
+     *             The camera plane is the plane of the camera view direction + camera
+     *             left direction (i.e. the negative camera right direction), with the
+     *             camera up direction as the normal. The angle goes from -pi to pi in
+     *             radians and zero degrees would be forward in the camera view direction.
+     *             When the object is located towards the left relative the camera view
+     *             direction, then the angle will be a positive value. If instead, the
+     *             object is located towards the right, the angle will become negative.
+     *
+     * Circular: Calculate the angle to the object in circular space around the center
+     *           point in the camera view plane. This mode is suitable for centered
+     *           fisheye displays or omnidirectional immersive environments (such as the
+     *           Hayden planetarium). The camera view plane is the plane of the camera up
+     *           direction + camera left direction (i.e. the negative camera right
+     *           direction), with the negative camera view direction as the normal. The
+     *           angle goes from -pi to pi in radians and zero degrees would be forward in
+     *           the camera view direction. When the object is located towards the left
+     *           relative the center of the screen, then the angle will be a positive
+     *           value. If instead, the object is located towards the right, the angle
+     *           will become negative.
+     */
+    enum class AngleCalculationMode {
+        Horizontal = 0,
+        Circular
+    };
+
+    TelemetryModule();
+    ~TelemetryModule();
+
+    /**
+     * Returns the Lua libraries for all telemetries available in the telemetry module
+     *
+     * \return The Lua libraries for all telemetries available in the telemetry module
+     */
+    std::vector<scripting::LuaLibrary> luaLibraries() const override;
+
+    virtual void internalInitialize(const ghoul::Dictionary& dictionary) override;
+    virtual void internalDeinitialize() override;
+
+    /**
+     * Get the list of telemetries that are currently registered in the module
+     *
+     * \return A list of all registered telemetries
+     */
+    const std::vector<TelemetryBase*>& telemetries() const;
+
+    /**
+     * Get a specified telemetry from the list of registered telemetries in the module
+     *
+     * \param id The identifier of the telemetry to fetch
+     *
+     * \return The requested telemetry
+     */
+    const TelemetryBase* telemetry(std::string id) const;
+    TelemetryBase* telemetry(std::string id);
+
+    /**
+     * Get the current angle calculation mode used in the telemetry module
+     *
+     * \return The angle calculation mode
+     */
+    AngleCalculationMode angleCalculationMode() const;
+
+    /**
+     * Return whether any elevation angles are being caclulated and sent over the osc
```

Comment:
```suggestion
     * Return whether any elevation angles are being caclulated and sent over the OSC
```

however, apparently it could be something else than OSC?